test/testutils.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
 
+// disable eyeglass deprecations until tests are rewritten without deprecated methods
+process.env.EYEGLASS_DEPRECATIONS = "false";
```

Comment:
This is a public API, right? I'm ok with having a default for it as set in the environment, but it should also have a corresponding option. It's ok to normalize this option first.